addons/point_of_sale/models/pos_order.py

```diff
@@ -348,10 +352,10 @@ def _default_pricelist(self):
     company_id = fields.Many2one('res.company', string='Company', required=True, readonly=True, default=lambda self: self.env.user.company_id)
     date_order = fields.Datetime(string='Order Date', readonly=True, index=True, default=fields.Datetime.now)
     user_id = fields.Many2one('res.users', string='Salesman', help="Person who uses the cash register. It can be a reliever, a student or an interim employee.", default=lambda self: self.env.uid)
-    amount_tax = fields.Float(compute='_compute_amount_all', string='Taxes', digits=0)
-    amount_total = fields.Float(compute='_compute_amount_all', string='Total', digits=0)
-    amount_paid = fields.Float(compute='_compute_amount_all', string='Paid', states={'draft': [('readonly', False)]}, readonly=True, digits=0)
-    amount_return = fields.Float(compute='_compute_amount_all', string='Returned', digits=0)
```

Comment:
In fact think we should commit further to this. Already you cannot create pos.order records in the backend. Also I don't think you can edit them but I'm not sure. In the end I don't think we should worry too much about the backend calculations at all. I feel like pos.order records should only be created through the frontend and should not be modifyable through the backend because they have already been paid anyway. I think we can consider removing ```_compute_amount_all``` and related functions all together.